src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/services/CamundaService.java

```diff
@@ -112,8 +115,8 @@ public Map<String, CamundaVariable> getTaskVariables(String taskId) {
 
     public boolean isCftTaskStateExistInCamunda(String taskId) {
         Map<String, Object> body = Map.of(
-            "variableName", CFT_TASK_STATE.value(),
-            "taskIdIn", singleton(taskId)
+            VARIABLE_NAME, CFT_TASK_STATE.value(),
+            TASK_ID_IN, singleton(taskId)
```

Comment:
Maybe we can have a similar convention here e.g. `KEY_VARIABLE_NAME` etc

Reply:
I made the changes as suggested Ravi.